setup.py

```diff
@@ -131,6 +131,7 @@
         "sentencepiece != 0.1.92",
         # for XLM
         "sacremoses",
+        "deberta",
```

Comment:
We can't do this unfortunately. Would you mind copy-pasting the code you need into `modeling_deeberta.py`

Reply:
+ reusing identical components of Roberta, including the config.

Reply:
Hi, while most of the components are functional the same the interface of ours' implementation is different from RoBERTa/BERT we also have some specially optimized components used inside of our implementation. Meanwhile we are going to continually improve our implementation. May I know the concern of adding reference to deberta?

Reply:
We don't have external deeplearning dependencies so that we can (a) fix things if they break and (b) provide one place where somebody can read all the model code they are using.

You can read more about the reasoning [here](https://huggingface.co/transformers/philosophy.html).

Reply:
Sure. We will merge the code into one single file.